packages/terra-menu/src/Menu.jsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popup from 'terra-popup';
+import SlideGroup from 'terra-slide-group';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import MenuItem from './MenuItem';
+import MenuItemGroup from './MenuItemGroup';
+import MenuDivider from './MenuDivider';
+import MenuContent from './_MenuContent';
+import MenuWidths from './_MenuWidths';
+import MenuUtils from './_MenuUtils';
+import styles from './Menu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * List of Menu.Items/Menu.ItemGroups to be displayed as content within the Menu.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Callback function indicating a close condition was met, should be combined with isOpen for state management.
+   */
+  onRequestClose: PropTypes.func.isRequired,
+  /**
+   * Target element for the menu to anchor to.
+   */
+  targetRef: PropTypes.func.isRequired,
+  /**
+   * Bounding container for the menu, will use window if no value provided.
+   */
+  boundingRef: PropTypes.func,
+  /**
+   * CSS classnames that are append to the arrow.
+   */
+  classNameArrow: PropTypes.string,
```

Comment:
This is copied from the popup api. Since it'll be passed through, I don't think it would make sense for the api's to differ